uclasm/matching/filters/topology_filter.py

```diff
@@ -0,0 +1,20 @@
+from ..local_cost_bound.edgewise import edgewise_local_costs
+import numpy as np
+
+def topology_filter(smp):
+    """Filtering based on topology.
+
+    TODO: Add the option of filtering just for one node
+
+    Parameters
+    ----------
+    smp : MatchingProblem
+        A subgraph matching problem on which to compute nodewise cost bounds.
+    """
+    disagreements = edgewise_local_costs(smp)
+
+    is_cand = np.logical_and(disagreements <= smp.global_cost_threshold,
```

Comment:
Just do a min between the two cost thresholds and compare to the lower of the two